app/models/common_names.py

```diff
@@ -0,0 +1,21 @@
+from typing import TYPE_CHECKING
```

Comment:
Can you consolidate all species related models in `species.py`?

Reply:
Ohh, okay you mean define multiple tables(classes) in one file only?

Reply:
Yes.

Reply:
Btw, we can do the same for crud and schemas.

Reply:
Yup, almost done with that.